### ISSUE TITLE
Add  option to allow sorting of the fields when merging types

### ISSUE THREAD

**Issue by WilliamHuang1995** [Urigo/merge-graphql-schemas#304]
First of all, what a great library.

I'd like to propose a feature request that allows the fields in a type to be sorted alphabetically.

That is, given two files:

```
type Query {
  A: String!
  C: String!
}
```
```
extend type Query {
  B: String!
  D: String!
}
```
the current result would merge the two graphql files into
```
type Query {
  A: String!
  C: String!
  B: String!
  D: String!
}
```
but in reality I would want 
```
type Query {
  A: String!
  B: String!
  C: String!
  D: String!
}
```
Thanks for reading!

**Comment by ardatan** [comment]
Implemented in https://github.com/ardatan/graphql-toolkit/commit/68e0cb4568613e67c0f1e96e330c791fc6afba8b

**Comment by ardatan** [comment]
Available in 1.7.4!
